samples/cifar10/README.md

```diff
@@ -1,16 +1,40 @@
-# Training simple network on CIFAR10 dataset
-## Download CIFAR10 dataset
+## Training convolution network on CIFAR10 dataset
+
+* Follow instruction to setup enviroment: [DeepWorks samples](../README.md)
+
+* Run train:
+```bash
+./samples/mnist/run_cifar10_train.sh
+```
+
+Possible output:
+```bash
+Epoch: 0
+Loss: 2.21671
+Loss: 2.04742
+Loss: 1.92269
+Loss: 1.91002
+Accuracy: 0.343249
+Model saved: build/cifar10_model.bin
```

Comment:
if you need it, i can give you more more extensive output with Accuracy after ~40 epochs, but a few hours later

like

```
Epoch: 0
Loss: 2.21671
Loss: 2.04742
Loss: 1.92269
Loss: 1.91002
Accuracy: 0.343249

...

Epoch: 26
Loss: 1.04711
Loss: 1.05685
Loss: 1.05886
Loss: 1.11385
Loss: 1.14189
Loss: 1.14818
Loss: 1.14877
Accuracy: +100500
```

Reply:
I believe it's OK for README.md but would be really interesting to know actual results